packages/bot-engine/src/getNextGroup.ts

```diff
@@ -141,6 +166,52 @@ export const getNextGroup = async ({
   };
 };
 
+export async function getNextBlockById(
```

Comment:
Is it possible to move this to executeResumeAfter file? Since it's only used there for now it can be a local function